servers/audio_server.cpp

```diff
@@ -1246,6 +1246,8 @@ void AudioServer::stop_playback_stream(Ref<AudioStreamPlayback> p_playback) {
 			_delete_stream_playback(p_playback);
 		}
 		return;
+	} else {
+		p_playback->stop(); // Hack solution to get _stop() to be called
 	}
```

Comment:
The fix is fine! It's not at all a hack solution. So, I suggest that you remove that comment and you don't have to put the code inside an else statement as there's a unilateral return inside the if.

```suggestion
	}

	p_playback->stop();
```

Reply:
Alright then, I'll make the change, amend the commit, and notify you

Reply:
Done and done!